Common/MathUtils/include/MathUtils/Chebyshev3D.h

```diff
@@ -27,7 +27,8 @@ class TMethodCall;  // lines 16-16
 
 
```

Comment:
why is this file changed / committed? Please only commit things related to your own developments.

Reply:
Sorry, I run clang and then did commit .
How to reverse this?

Reply:
Since it is in your last commit, do:
cd O2
git reset HEAD~1
git checkout Common/Field/include/Field/MagneticWrapperChebyshev.h
git checkout Common/MathUtils/include/MathUtils/Chebyshev3D.h 
git checkout Common/MathUtils/include/MathUtils/Chebyshev3DCalc.h
git checkout Common/MathUtils/include/MathUtils/MathBase.h
git checkout  Detectors/TPC/calibration/src/CalibPulser.cxx

git commit -a -m "MC labels implemented; bug in optical properties fixed"

git push -f AllaMaevskaya fit-common-v0

How are you applying the clang-format?

Reply:
git clang-format HEAD~10

Reply:
Why 10? You had only 4 commits